config/routes.rb

```diff
@@ -5,15 +5,37 @@
 
   # You can have the root of your site routed with "root"
   root 'tasks#index'
-  # same as get '/' => "task_lists_rails#index"
+  # same as get '/' => "tasks#index"
 
   # Example of regular route:
   # get  'products/:id'   => 'catalog#view'
-  get    '/tasks/new'         => 'tasks#new'
-  post   '/tasks'             => 'tasks#create'
-  delete '/tasks/delete/:id'  => 'tasks#destroy'
-  patch  '/tasks/update/:id'  => 'tasks#update'
-  get    '/tasks/:id'         => 'tasks#show'
+
+  # Display people
+  get     '/people'             => 'people#index'
+
+  # Display person and their tasks
+  get     '/people/:id'         => 'people#show'
+  get     '/people/:id/tasks'   => 'people#person_tasks'
+
+  # Create Tasks
+  get     '/tasks/new'          => 'tasks#new'
+  post    '/tasks'              => 'tasks#create'
+
+  # Display task
+  get     '/tasks/:id'          => 'tasks#show'
+
+  # Delete task
+  delete  '/tasks/delete/:id'   => 'tasks#destroy'
```

Comment:
In this case you could actually skip the "delete" in the path because you are using the `delete` method which makes this a unique action for the URL. So you could just use:

```ruby
delete '/tasks/:id' => 'tasks#destroy'
```